modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+By configuring delegated image scanning, you can scan images by using the secured clusters instead of the Central services.
+
+A new delegated registry configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries. 
+
+To enable delegation of scans by using the `roxctl` CLI, Jenkins plugin, or API, you must also specify a destination cluster and source registry.
+
+.Prerequisites
+
+* You have installed Scanner-slim in the secured cluster to scan images.
++
+[NOTE]
+====
+Enabling Scanner-slim is supported on {ocp} and Kubernetes secured clusters.
+====
+
+* You have set up image mirroring for registries.
++
+For more information, see "Enabling seamless image access and efficient use of multiple registry sources".
+
+.Procedure
+
+. In the {product-title-short} portal, click *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the following information:
+
+** *Delegate scanning for*: To choose the scope of the image delegation, select one of the following options:
+*** *None*: The default option. This option specifies that the secured clusters do _not_ scan any images, except for images from the {ocp} integrated registry.
+*** *All registries*: This option indicates that the secured clusters scan all the images.
+*** *Specified registries*: This option specifies the images that secured clusters should scan based on the registries list.
+
+** *Select default cluster to delegate to*: From the drop-down list, select the name of the default cluster. The default cluster processes the scan requests coming from the command-line interface (CLI) and API. This is optional and you can select `None` if required.
+
+** Optional: To specify the source registry and destination cluster details, click *Add registry*. 
```

Comment:
Could this be raised up a level and made into an ordered list item?

Reply:
No, there is no particular sequence to do these steps. So I would prefer to keep it as an unordered list.